### PR TITLE
Pin docker image versions

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
fixes https://github.com/mitodl/odl-video-service/issues/692

#### What's this PR do?
Pins nginx, postgres and redis docker images

#### How should this be manually tested?
Nothing should break
if you are getting db issue then
- `docker-compose kill db`
- `docker-compose rm db`
-  `docker-compose up`

@pdpinch

### CODE REVIEW

**Comment by noisecapella** [comment]
Looks like there's a pylint bug somewhere... can you try changing the version to match open-discussions or micromasters?

**Comment by amir-qayyum-khan** [comment]
@noisecapella these are exact same versions

**Comment by amir-qayyum-khan** [comment]
@noisecapella astroid is giving issues with pylint

File: test_requirements.txt [hunk 1]
```diff
@@ -6,8 +6,8 @@ ipdb
 nplusone
 pdbpp
 pip-tools
-pylint==1.7.1
-pylint-django==0.7.2
+pylint==1.8.2
```
Comment (mbertrand) [inline test_requirements.txt]: Using these versions should fix it (same as for open-discussions):
```
pylint==1.7.1
pylint-django==0.7.2
astroid==1.5.3
```

**Comment by amir-qayyum-khan** [comment]
Done @noisecapella

**Comment by noisecapella** [comment]
@amir-qayyum-khan Can you change the redis version to 3.2.10? That's what we're using on production

**Comment by amir-qayyum-khan** [comment]
@noisecapella in docker-compose-travis.yml or requirments.txt?

**Comment by amir-qayyum-khan** [comment]
Done @noisecapella @mbertrand

**Pull request merged**